sdk/src/Services/S3/Custom/Transfer/Internal/_async/BaseCommand.async.cs

```diff
@@ -83,7 +83,7 @@ await completedTask
             }
         }
 
-        protected static async Task ExecuteCommandAsync<T>(BaseCommand<T> command, CancellationTokenSource internalCts, SemaphoreSlim throttler) where T : class
```

Comment:
assuming this is safe because uploaddirectory/downloaddirectory are the only commands that use this function signature